README.rst

```diff
@@ -154,25 +151,6 @@ will need to be installed. In Ubuntu-based distributions, this can be done as fo
 .. code-block:: console
 
     sudo apt-get install cmake libgtest-dev
-    cd /usr/src/googletest/googletest
-    sudo cmake
-    sudo make
-    sudo cp libgtest* /usr/lib/
-    sudo mkdir /usr/local/lib/googletest
-    sudo ln -s /usr/lib/libgtest.a /usr/local/lib/googletest/libgtest.a
-    sudo ln -s /usr/lib/libgtest_main.a /usr/local/lib/googletest/libgtest_main.a
```

Comment:
This was due to a bug in Ubuntu < 20.04 --- they would install googletest in the wrong place! Hence the need to make the symlinks.

Reply:
Should we keep it? I definitely do not have such new version :P

Reply:
Oh, maybe it was only present in Ubuntu < 18.04.. I had this bug before I upgraded 😆 

I think it's fine to delete it 👍